.github/workflows/ci.yaml

```
@@ -0,0 +1,38 @@
+  name; CI
+  on:
+    pull_request:
+    workflow_dispatch:
+    push:
+      branches:
+        - main
+  jobs:
+    CI:
+      runs-on: ubuntu-latest
+      permissions:
+        id-token: write
+        contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+        role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+        aws-region: eu-west-1
+      - uses: actions/setup-java@v3
+        with:
+          java-version: "11"
+          distribution: "corretto"
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
```

Could we source the node version from a `.nvmrc` file? Else we'd have to remember to keep the two in-sync.

---

Non-blocking: update this to an `.nvmrc` node version file!

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: ".nvmrc"
```